recipes/libcurl/all/conanfile.py

```diff
@@ -153,6 +153,12 @@ def config_options(self):
             del self.options.with_zstd
         if not self._has_metalink_option:
             del self.options.with_libmetalink
+
+        # Before 7.86.0, enabling unix sockets configure option would fail on windows
+        # It was fixed with this PR: https://github.com/curl/curl/pull/9688
+        if self._is_mingw and Version(self.version) < "7.86.0":
```

Comment:
```suggestion
        if (self._is_mingw and Version(self.version) < "7.86.0") or self._is_using_cmake_build:
```

Reply:
because this option is not even used in CMake build currently (like several other options, but out of scope of this PR)

Reply:
Should I apply it anyway or does that need to come from a different PR?

Reply:
In another PR it's fine